release-schema.json

```diff
@@ -13,8 +13,54 @@
         },
         "legalBasis": {
           "title": "Legal basis",
-          "description": "The legal basis of the procedure.",
-          "$ref": "#/definitions/Classification"
+          "description": "The laws and regulations that govern the contracting process and grant legal authority to the procuring entity.",
+          "type": "object",
+          "properties": {
+            "scheme": {
+              "title": "Scheme",
+              "description": "The identifier scheme from which the legislation's identifier is taken.",
+              "type": [
+                "string",
+                "null"
+              ],
+              "codelist": "itemClassificationScheme.csv",
+              "openCodelist": true,
+              "minLength": 1
+            },
+            "id": {
+              "title": "ID",
+              "description": "The legislation's identifier taken from the scheme.",
+              "type": [
+                "string",
+                "null"
+              ],
+              "minLength": 1
+            },
+            "description": {
+              "title": "Description",
+              "description": "A textual description or title for the legislation's identifier.",
+              "type": [
+                "string",
+                "null"
+              ],
+              "minLength": 1
+            },
+            "uri": {
+              "title": "URI",
+              "description": "A URI to uniquely identify the legislation.",
+              "type": [
+                "string",
+                "null"
+              ],
+              "format": "uri"
+            },
+            "wasDerivedFrom": {
+              "title": "Was derived from",
+              "description": "The law or regulation that the legislation transposes, updates or builds upon. For example, in the EU, some national legislation transposes EU directives.",
+              "$ref": "#/definitions/Classification"
```

Comment:
Instead of using Classification (which ends up with a very heterogenous codelist), create a Legislation definition with the same four fields as for legalBasis, using the new codelist for `scheme`.